### PR TITLE
Feat/token info page

### PR DESCRIPTION
**Pull request by haroonpsyoptions**

### CODE REVIEW

**Comment by haroonpsyoptions** [comment]
image
image
image
image
image
image

**Review by tomjohn1028 — CHANGES_REQUESTED** [review]
@haroonpsyoptions is looks like you implemented this into the landing page. The Token page should be a separate page. Then there should be an added nav bar link on the landing page to link to the token page you implemented

Screen Shot 2022-03-02 at 2 18 42 PM

None of the images are loading as well.

File: src/styles/Token.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .psyoptions-token {
   .p-content img {
-    width: 100%;
+    // width: 100%;
```
Comment (Taylor123) [inline src/styles/Token.scss:3]: nit: should remove if we're going to leave this commented

**Comment by Taylor123** [comment]
How come the package-lock.json is deleted?

File: src/styles/Token.scss (hunk 1 above)
Comment (haroonpsyoptions) [reply src/styles/Token.scss:3]: > How come the package-lock.json is deleted?

What do you mean?

Comment (tomjohn1028) [reply src/styles/Token.scss]: @haroonpsyoptions if a comment does not add any value to another dev in the codebase then it should not exist. I have removed the comment and the unnecessary scss file

**Review by tomjohn1028 — APPROVED** [review]

**Pull request merged**